src/app/clusters/general-commissioning-server/general-commissioning-server.cpp

```diff
@@ -239,34 +257,63 @@ bool emberAfGeneralCommissioningClusterCommissioningCompleteCallback(
         }
         else
         {
-            if (failSafe.NocCommandHasBeenInvoked())
+            CHIP_ERROR err;
+
+#if CHIP_CONFIG_TC_REQUIRED
+
+            uint16_t termsAndConditionsAcceptedAcknowledgements;
+            bool hasRequiredTermAccepted;
+            bool hasRequiredTermVersionAccepted;
```

Comment:
Always give default values to variables expected to be set by reference passing to methods. Set these to false.